package.json

```diff
@@ -83,6 +92,8 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "^2.2.0",
+    "ember-cli-typescript": "^2.0.0-beta.1",
```

Comment:
We are very serious about this being a pre-release version. Please assume that it could break at any time, and take on acceptable risk accordingly

Reply:
playing with potentially breaking things is my life ;)